pythonforandroid/bootstraps/webview/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -78,6 +78,8 @@
             {%- if args.intent_filters -%}
             {{- args.intent_filters -}}
             {%- endif -%}
+            {% for m in args.activity_meta_data %}
```

Comment:
Suffix the Jinja block with `{%-` and it will strip the whitespace before it. Then you can put the `endif` directive on a newline with the same and it will come out correctly. Although, the previous line should probably not do `-%}`.